agent/src/beerocks/slave/son_slave_thread.h

```diff
@@ -248,6 +248,7 @@ class slave_thread : public beerocks::socket_thread {
     bool handle_multi_ap_policy_config_request(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_rx);
     bool handle_client_association_request(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_rx);
     bool handle_client_steering_request(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_rx);
+    bool handle_beacon_metrics_query_request(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_rx);
```

Comment:
Any reason to not pass the `cmdu_rx` as a const reference?
I know that other functions do not, but maybe they should...

Reply:
Indeed. sometimes there is a tense between doing things _correctly_ vs. _uniformly_.
The tense rises from the fact that both have positive implications.
The "spirit" that I got from this project up till now is that uniform wins.